Project-I-Social-Card/social-card/src/components/CardComponents/Card.css

```diff
@@ -1 +1,18 @@
 /* All styles for your Card Components live here */
+
+.card-content a {
+    text-decoration: none;
+    color: lightgray;
+    
```

Comment:
The `.card-content` component also has a gray rounded `border` at its right, bottom and left sides.